package.json

```diff
@@ -65,7 +65,10 @@
     "parse5": ">= 1.3.1 < 2.0.0",
     "request": ">= 2.44.0 < 3.0.0",
     "xml-name-validator": "^1.0.0",
-    "xmlhttprequest": ">= 1.6.0 < 2.0.0"
+    "xmlhttprequest": ">= 1.6.0 < 2.0.0",
+    "acorn-globals": "^1.0.2",
+    "acorn": "~0.11.0",
```

Comment:
Pin to 0.11.0 exactly until https://github.com/marijnh/acorn/issues/209 is fixed